lnwire/features.go

```diff
@@ -129,6 +129,16 @@ const (
 	// transactions, which also imply anchor commitments.
 	AnchorsZeroFeeHtlcTxOptional FeatureBit = 23
 
+	// AMPRequired is a required feature bit that signals that the receiver
```

Comment:
Bits in this commit no longer match the commit title (4ca2eee8dfbfb96d06a8a42b2809b392fe049faf): commit title references bit 24/25, while this is using 30/31 now.